homework/Vasili_Zasinets/src/main/java/homework/shop/Shop.java

```diff
@@ -0,0 +1,41 @@
+package homework.shop;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+
+public class Shop {
+    Logger log = LoggerFactory.getLogger(" ");
+    LinkedHashMap<String, BigDecimal> myShop = new LinkedHashMap<>();
+    Product myProduct;
+
+    @Override
+    public String toString() {
+        return " " +
+                "shop =" + myShop +
+                '}';
+    }
+
+    void addProduct(String name, BigDecimal price) {
+        myShop.put(name, price);
+
+    }
```

Comment:
if you adding product, than it will be expected to pass Product into the method.

Reply:
fixed